src/main/java/org/team1540/robot2019/subsystems/Drivetrain.java

```diff
@@ -0,0 +1,390 @@
+        driveLeftMotorA
+            .set(ControlMode.Velocity, velocity, DemandType.ArbitraryFeedForward, feedForward);
+    }
+
+    public void setRightVelocityTicks(double velocity, double feedForward) {
+        configTalonsForVelocity();
+        driveRightMotorA
+            .set(ControlMode.Velocity, velocity, DemandType.ArbitraryFeedForward, feedForward);
+    }
+
+    public void setLeftVelocityMetersPerSecond(double velocity) {
+        setLeftVelocityTicks(velocity / 10 * Tuning.drivetrainTicksPerMeter);
+    }
+
+    public void setRightVelocityMetersPerSecond(double velocity) {
+        setRightVelocityTicks(velocity / 10 * Tuning.drivetrainTicksPerMeter);
+    }
+
+    public void setPercent(double left, double right) {
+        setLeftPercent(left);
+        setRightPercent(right);
+    }
+
+    public void setLeftPercent(double percent) {
+        driveLeftMotorA.set(ControlMode.PercentOutput, percent);
+    }
+
+    public void setRightPercent(double percent) {
+        driveRightMotorA.set(ControlMode.PercentOutput, percent);
+    }
+
+    public void setBrake(boolean brake) {
+        for (ChickenController talon : driveMotorMasters) {
+            talon.setBrake(brake);
+        }
+    }
+
+    public double getLeftPositionTicks() {
+        return driveLeftMotorA.getSelectedSensorPosition();
+    }
+
+    public double getRightPositionTicks() {
+        return driveRightMotorA.getSelectedSensorPosition();
+    }
+
+    public double getLeftPositionMeters() {
+        return getLeftPositionTicks() / Tuning.drivetrainTicksPerMeter;
+    }
+
+    public double getRightPositionMeters() {
+        return getRightPositionTicks() / Tuning.drivetrainTicksPerMeter;
+    }
+
+    public double getLeftVelocityTicks() {
+        return driveLeftMotorA.getSelectedSensorVelocity();
+    }
+
+    public double getRightVelocityTicks() {
+        return driveRightMotorA.getSelectedSensorVelocity();
+    }
+
+    public double getLeftVelocityMetersPerSecond() {
+        return getLeftVelocityTicks() * 10 / Tuning.drivetrainTicksPerMeter;
+    }
+
+    public double getRightVelocityMetersPerSecond() {
+        return getRightVelocityTicks() * 10 / Tuning.drivetrainTicksPerMeter;
+    }
+
+    public Twist2D getTwist() {
+        double xvel = (getLeftVelocityMetersPerSecond() + getRightVelocityMetersPerSecond()) / 2;
+        double thetavel = (getLeftVelocityMetersPerSecond() - getRightVelocityMetersPerSecond()) / (Tuning.drivetrainRadiusMeters) / 2;
+        return new Twist2D(xvel, 0, thetavel);
+    }
+
+    public double getLeftPercent() {
+        return driveLeftMotorA.getMotorOutputPercent();
+    }
+
+    public double getRightPercent() {
+        return driveRightMotorA.getMotorOutputPercent();
+    }
+
+    public void zeroEncoders() {
+        Hardware.driveLeftMotorA.setSelectedSensorPosition(0);
+        Hardware.driveRightMotorA.setSelectedSensorPosition(0);
+    }
+
+    public void stop() {
+        setLeftVelocityTicks(0);
```

Comment:
This is unintuitive; I'd expect a `stop` to stop the motors not make them PID somewhere.